.cirrus.yml

```diff
@@ -208,12 +208,12 @@ task:
         matrix:
           CHANNEL: "master"
           CHANNEL: "stable"
-      install_script:
+      install_background_script:
         - git clone https://github.com/flutter/web_installers.git
         - cd web_installers/packages/web_drivers/
         - dart pub get
         - dart lib/web_driver_installer.dart chromedriver --install-only
-        - ./chromedriver/chromedriver --port=4444 &
+        - ./chromedriver/chromedriver --port=4444
```

Comment:
We probably want this to be a separate script step that is background, and the rest to stay as-is.

Reply:
```
      install_script:
        - git clone https://github.com/flutter/web_installers.git
        - cd web_installers/packages/web_drivers/
        - dart pub get
        - dart lib/web_driver_installer.dart chromedriver --install-only
      install_background_script:
        - ./chromedriver/chromedriver --port=4444
```

?

Reply:
See 02dac56e0d663693c813aee7e11c7ae66791371c